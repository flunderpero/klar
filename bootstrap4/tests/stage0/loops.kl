fn main():
    mut i = 0
    loop:
        i = i + 1
        if i == 3 => break
        print(i)
    end
    loop:
        i = i + 1
        if i == 3 => continue
        if i == 6 => break
        print(i)
    end
    return 0
end
