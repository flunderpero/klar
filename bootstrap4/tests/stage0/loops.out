1
2
4
5
---
exit code: 0
